classes/01class/exercises/c01-e04/drii-cavalcanti/c01-e04.txt

```diff
@@ -0,0 +1,15 @@
+Create a S3 bucket:
+S3 -> Create a bucket
+Bucket name: c01-e04-drii-cavalcanti
+Choose: c01-e04-drii-cavalcanti -> Upload -> Add File
+
+Linux Terminal:
+ssh -i "<my key pair>.pem" ubuntu@ec2-18-233-100-104.compute-1.amazonaws.com
+ubuntu@ip-172-31-88-249:sudo apt install awscli | aws configure (using my previous accessKeys)
```

Comment:
For the c01-e04 exercise, you shouldn't use your aws key/secret on your EC2 instances. AWS has another feature called Instance Profiles where you can add permissions to your EC2 instance to access S3.

Can you check the links below and try to do the exercise again using Instance Profiles instead of your aws key/secret?

https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html
https://docs.aws.amazon.com/codedeploy/latest/userguide/getting-started-create-iam-instance-profile.html